brat/Brat/Checker/Monad.hs

```diff
@@ -139,29 +139,29 @@ throwLeft :: Either ErrorMsg a -> Checking a
 throwLeft (Right x) = pure x
 throwLeft (Left msg) = err msg
 
-vlup :: UserName -> Checking [(Src, BinderType Brat)]
+vlup :: QualName -> Checking [(Src, BinderType Brat)]
```

Comment:
Oooh, that's odd. We can have qualified names of variables?

Reply:
Ah right yes thank you. Ok - so `vlup` is really the misnamed one - it does not necessarily look up variables; it just looks up names. Really should be `nlup` or `qnlup` or just `lup` (or `blup` to distinguish from `klup`? Hey, that one's actually pronouncable in one syllable... 😁 😀 )